src/database/entities/payment.account.entity.ts

```diff
@@ -8,6 +8,7 @@ import type { KeyEntity } from './key.entity.js';
 dotenv.config();
 
 @Entity('paymentAccount')
+@Index(['customer', 'namespace'], { unique: true })
```

Comment:
Looks like generated migrations are not saved under `src/database/migrations`, if one runs `npm run generate`.

Could you please resolve, generate and run migrations on staging?

I do remember migrations being run within DO. If that's the case, skip running them manually.

Reply:
Added the migration script